uffizzi/DockerfileDB

```diff
@@ -0,0 +1,3 @@
+FROM postgres:12-alpine
+
+COPY ./sql/schema.postgresql.sql /docker-entrypoint-initdb.d/schema.postgresql.sql
```

Comment:
very cool using this feature of the image! I wonder if the new "non-empty volume" feature can do this so you don't need to build an image?